src/components/Hero/Hero.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import heatmapPNG from '../../images/heatmap.png';
+import * as S from './Hero.style';
+
+const Hero = () => (
+  <S.Wrapper>
+    <S.Content>
+      <S.Title>No reactions to your reddit posts?</S.Title>
+      <S.Subtitle>
+        Great timing, great results! Find the best time to post on your
+        subreddit.
+      </S.Subtitle>
+      <Link to="/search/javascript">
+        <S.CTALink>Show me the best time</S.CTALink>
+      </Link>
+      <S.Small>r/javascript</S.Small>
+    </S.Content>
+    <section>
+      <Link to="/search/javascript">
+        <S.Image
+          src={heatmapPNG}
+          alt="heatmap"
+          srcSet="/heatmap.png, /heatmap@2x.png 2x, /heatmap@3x.png 3x"
+        />
+      </Link>
+    </section>
+    <S.Info>
```

Comment:
Ah now I understand why there isn't a separate info-section PR 😄

Reply:
😄  yea I got ahead of myself again, I assumed the hero section and info section were combined into one task lol my mistake